homeassistant/components/pi_hole/__init__.py

```diff
@@ -55,6 +56,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: PiHoleConfigEntry) -> bo
     verify_tls = entry.data[CONF_VERIFY_SSL]
     location = entry.data[CONF_LOCATION]
     api_key = entry.data.get(CONF_API_KEY, "")
+    version = entry.data.get(CONF_API_VERSION, 5)
```

Comment:
I've chosen version 5 here because if there is not a value then we can assume a legacy setup?